.env.template

```diff
@@ -6,4 +6,4 @@ RSPEC_RETRY_RETRY_COUNT=0
 MAILER_SENDER="Rails World <hello@teloslabs.co>"
 REGISTRATION_ENABLED=true
 SESSION_REMINDERS_ENABLED=true
-ONLY_TESTER_REGISTRATION_ENABLED=false
\ No newline at end of file
+ONLY_TESTER_REGISTRATION_ENABLED=false
```

Comment:
needed empty line at the end of the file because bin/setup was addinga variable in the same line